orchagent/fdborch.h

```diff
@@ -11,7 +11,14 @@ struct FdbEntry
     sai_vlan_id_t vlan;
     bool operator<(const FdbEntry& other) const
     {
-        return tie(mac, vlan) < tie(other.mac, other.vlan);
+        // operator<(const MacAddress&) works incorrectly for some reason
```

Comment:
Restored previous code. The issue was fixed there: https://github.com/Azure/sonic-swss-common/pull/93